packages/KPagination/KPagination.vue

```diff
@@ -0,0 +1,328 @@
+<template>
+  <div
+    v-if="totalCount > currentPageSize"
+    class="card-pagination-bar">
+    <span class="pagination-text">
+      <span class="pagination-text-pages">{{ pagesString }}</span>
+      {{ pageCountString }}
+    </span>
+    <div class="pagination-button-container">
+      <button
+        :class="{ disabled: backDisabled }"
+        class="pagination-button square"
+        @click="pageBack"
+      >
+        <KIcon
+          :color="backDisabled ? 'var(--grey-500)' : 'var(--blue-400)'"
+          icon="arrowLeft"
+          width="16px"
+          heigth="16px"
+          view-box="0 0 16 14"
+        />
+      </button>
+      <button
+        v-if="firstDetached"
+        class="pagination-button"
+        @click="changePage(1)"
+      >
+        1
+      </button>
+      <button
+        v-if="firstDetached"
+        class="pagination-button placeholder">
+        ...
+      </button>
+      <button
+        v-for="page in pagesVisible"
+        :key="page"
+        :class="{ active: page == currentPage }"
+        class="pagination-button"
+        @click="changePage(page)"
+      >
+        {{ page }}
+      </button>
+      <button
+        v-if="lastDetached"
+        class="pagination-button placeholder">
+        ...
+      </button>
+      <button
+        v-if="lastDetached"
+        class="pagination-button"
+        @click="changePage(pageCount)"
+      >
+        {{ pageCount }}
+      </button>
+      <button
+        :class="{ disabled: forwardDisabled }"
+        class="pagination-button square"
+        @click="pageForward"
+      >
+        <KIcon
+          :color="forwardDisabled ? 'var(--grey-500)' : 'var(--blue-400)'"
+          icon="arrowRight"
+          width="16px"
+          heigth="16px"
+          view-box="0 0 16 14"
+        />
+      </button>
+    </div>
+    <span class="page-size-select">
+      <KSelect
+        :placeholder="`${currentPageSize} items per page`"
+        :items="pageSizeOptions"
+        appearance="button"
+        @selected="updatePageSize"
+      />
+    </span>
+  </div>
+</template>
+
+<script>
+import KIcon from '@kongponents/kicon/KIcon.vue'
+import KSelect from '@kongponents/kselect/KSelect.vue'
+
+export default {
+  name: 'KPagination',
+  components: {
+    KIcon,
+    KSelect
+  },
+  props: {
+    items: {
+      type: Array,
+      default: () => []
+    },
+    totalCount: {
+      type: Number,
+      default: 0
+    },
+    pageSizes: {
+      type: Array,
+      default: () => [15, 25, 50, 75, 100]
+    },
+    neighbours: {
```

Comment:
```suggestion
    neighbors: {
```

We need to use US spelling I think...